packages/scxa-tsne-plot/src/ClusterTSnePlot.js

```diff
@@ -263,10 +203,8 @@ ClusterTSnePlot.propTypes = {
   onChangeColourBy: PropTypes.func,
   clusterType: PropTypes.string,
 
-  perplexities: PropTypes.arrayOf(PropTypes.number).isRequired,
-  selectedPerplexity: PropTypes.number.isRequired,
-  onChangePerplexity: PropTypes.func.isRequired,
-
+  selectedPlotType: PropTypes.string.isRequired,
+  selectedParameter: PropTypes.number.isRequired,
   onChangePlotOptions: PropTypes.func.isRequired,
   onChangePlotTypes: PropTypes.func.isRequired,
```

Comment:
These props are not being used either so it would be good to remove them if they are not required.